src/js/index.js

```diff
@@ -0,0 +1,27 @@
+const submit = e => {
+  e.preventDefault();
+  let input = document.querySelector('input');
+  if(input.value != '') {
+    addTask(input.value);
+  }
+}
+
+const addTask = newValue => {
+  let ul = document.querySelector('ul');
+  let newLi = document.createElement('li')
+  newLi.classList.add('box');
+  newLi.textContent = `${newValue}`;
```

Comment:
why using template strings?
```suggestion
  newLi.textContent = newValue;
```